tests/dialects/test_mssql_specific.py

```diff
@@ -226,7 +226,7 @@ def test_constraint_unique():
                         "references": None,
                         "size": None,
                         "type": "BIGINT",
-                        "unique": True,
+                        "unique": False,
```

Comment:
All these changes were from the move to handle compound unique clauses/statements.